arbos/block_processor.go

```diff
@@ -288,6 +313,22 @@ func ProduceBlockAdvanced(
 				return nil, nil, err
 			}
 
+			if chainConfig.ArbitrumChainParams.EnableEspresso {
```

Comment:
This is for a feature we called "builder guaranteed transactions". It allows wallets to "tag" their transactions with an Ethereum address of a builder to only allow that builder to include the transaction in a block.